src/elements/content-sidebar/BoxAISidebarContent.tsx

```diff
@@ -59,6 +72,14 @@ function BoxAISidebarContent(props: ApiWrapperProps) {
         setCacheValue('questions', questions);
     }
 
+    const onModalClose = () => {
+        setIsModalOpen(false);
+    };
+
+    const onSwitchToModalClick = () => {
+        setIsModalOpen(true);
+    };
```

Comment:
nit: these could be named as `handleModalClose` and `handleModalOpen`

reserving `on` for the prop and `handle` for the function